lib/pages/chat/chat_view.dart

```diff
@@ -127,7 +127,7 @@ class ChatView extends StatelessWidget with MessageContentMixin {
     }
 
     return GestureDetector(
-      onTapDown: controller.setReadMarker,
+      onTapDown: (_) => controller.setReadMarker(),
```

Comment:
please create ticket to improve `setReadMarker`

Reply:
dont' forget it

Reply:
Done #1147